benchmarks/static_test/test_discovery.py

```diff
@@ -5,16 +5,12 @@
 integration test framework to run benchmark executions.
```

Comment:
I think the rename of `benchmarks/test` to `benchmarks/static_test` also breaks Salvo. There is reference to the path in its configuration.

I would recommend we try to avoid changing the pre-existing layout and target names, since the `/benchmarks` subdirectory technically is public and part of Nighthawk's API.

Reply:
shifted static_test, targets, etc to prior.